bin/install-theos

```diff
@@ -144,11 +144,10 @@ get_sdks() {
 		update "SDKs appear to already be installed."
 	else
 		update "SDKs do not appear to be installed. Installing now..."
-		curl -L https://api.github.com/repos/theos/sdks/tarball -o sdks.tar.gz
 		TMP=$(mktemp -d)
-		tar -xvf sdks.tar.gz --strip=1 -C $TMP
+		curl -sL https://api.github.com/repos/theos/sdks/tarball | tar -xzvf - --strip=1 -C $TMP
```

Comment:
nit: can we `-C` directly into `$THEOS/sdks` and remove the need for `$TMP` entirely?

Reply:
I think it's helpful because the script only moves the `.sdk` items as opposed to all the scripts and other files in the archive

Reply:
Just tested - using `tar xf - --strip=1 -C "${THEOS}/sdks/" "*.sdk"` worked well for me

Reply:
```
tar: Option --wildcards is not supported
```

It appears the flag does not exist in BSD `tar`

Reply:
While have less intermediates would be nice, perhaps more performance and even robust, given this discrepancy I feel more comfortable keeping the current `mktemp` and `mv` than trying to support both `tar` commands

Reply:
Fun. How do we feel about a one-liner?
```
tar --version | grep GNU &> /dev/null && <tar cmd w/ --wildcards> || <tar cmd w/o --wildcards>
```

Reply:
haha, no worries. that `tar --version` is fairly simple. I'm also good with either.

Attaching macOS 13.5.2 `tar` output for reference:

```
bsdtar 3.5.3 - libarchive 3.5.3 zlib/1.2.11 liblzma/5.0.5 bz2lib/1.0.8
```

Reply:
Resolving as superseded by https://github.com/theos/theos/commit/1cfe85938ee26cac704f2e03f25ab114c5bfaa2f.